src/Sylius/Bundle/CoreBundle/CatalogPromotion/Command/RemoveInactiveCatalogPromotion.php

```diff
@@ -13,9 +13,19 @@
 
 namespace Sylius\Bundle\CoreBundle\CatalogPromotion\Command;
 
+/** @deprecated since 1.12 and will be removed in Sylius 2.0. Use {@see RemoveCatalogPromotion} instead. */
 final class RemoveInactiveCatalogPromotion
 {
     public function __construct(public string $code)
     {
+        trigger_deprecation(
+            'sylius/core-bundle',
+            '1.12',
+            sprintf(
+                'The "%s" class is deprecated since Sylius 1.12 and will be removed in 2.0. Use "%s" instead.',
+                self::class,
```

Comment:
```suggestion
            '1.13',
            sprintf(
                'The "%s" class is deprecated since Sylius 1.13 and will be removed in 2.0. Use "%s" instead.',
                self::class,
```